### PR TITLE
upgpkg: mivisionx 5.1.0-1

### PR DESCRIPTION
**Pull request by JP-Ellis**
MIVisionX does not yet work with FFMpeg 5.0; however FFMpeg 4.4 is available from the Arch repositories (it is a dependency for Firefox, OpenCV, VLC, Blender, among other).

Fixes: #429
Fixes: #597
Patches: #677

### CODE REVIEW

**Review by acxz — CHANGES_REQUESTED** [review]
Thanks for the PR @JP-Ellis 

Can you check if the binaries for mivisionx are properly installed/linked?

See: https://github.com/rocm-arch/rocm-arch/pull/666/files#diff-9e6a3bf1de3e79e6f7f2c1b04f847b47ab02327d5f1d443dece08aa8d8a1ac66R40-R41

```
    local _fn
  - for _fn in mivisionx; do
  + for _file in bin/*; do
  +  _fn="$(basename -- $_file)"
      ln -s "/opt/rocm/mivisionx/bin/$_fn" "$pkgdir/usr/bin/$_fn"
    done
```

**Comment by JP-Ellis** [comment]
Sorry for the delay.  It's all done now and all generated binaries are correctly moved over.

**Review by acxz — APPROVED** [review]
thanks for iterating!

**Comment by JP-Ellis** [comment]
I have pushed another commit to upgrade to `5.1.0`.

**Comment by acxz** [comment]
@JP-Ellis  can you squash the commits into one real quick?

**Comment by JP-Ellis** [comment]
Done :+1:

File: mivisionx/PKGBUILD
Comment (tpkessler) [inline mivisionx/PKGBUILD:2]: Could you please add me as co-maintainer?

Comment (acxz) [reply mivisionx/PKGBUILD:2]: Technically it should be the following:

Comment (acxz) [inline mivisionx/PKGBUILD:2]:
```suggestion
# Maintainer: Torsten Keßler <t dot kessler at posteo dot de>
# Contributor: acxz <akashpatel2008 at yahoo dot com>
# Contributer: JP-Ellis <josh@jpellis.me>
```

**Comment by acxz** [comment]
@tpkessler can you merge this as is?
It's been sitting for a while, we can worry about the 5.1.1 release for this pkg after this is merged.

**Comment by tpkessler** [comment]
Yes, I'm working on. I'm currently trying to build the new LLVM package in a clean chroot. Once I have built miopen, I'll merge this one.

**Pull request merged**
